docs/user-guide/cli-scsplugin.md

```diff
@@ -0,0 +1,91 @@
+# Secure Credential Store Plug-in for Zowe CLI
```

Comment:
Don't forget to add the `&trade` symbol after the name Zowe
Same thing may apply to other tiles (Like FTP for example)